src/openms/include/OpenMS/ANALYSIS/ID/SiriusMSConverter.h

```diff
@@ -55,7 +56,13 @@ namespace OpenMS
 
     /// store MS file
     /// @return string (full path to file)
-    static void store(const PeakMap & spectra, const OpenMS::String & msfile);
+    
+    // spectra: Peakmap from input mzml
+    // msfile: (internal) written .ms file from sirius 
+    // map_precursor_to_adducts: adducts of a spectrum (index). 
+    // Adducts are written to SIRIUS ms file. If adduct information for a spectrum is missing, 
+    // no adduct information is written. In this case, SIRIUS assumes default adducts for the respective spectrum.
+    static void store(const PeakMap & spectra, const String & msfile, const std::map<size_t, StringList> & map_precursor_to_adducts);
```

Comment:
@oliveralka could you use the standard comments that also doxygen understands, see for example https://github.com/OpenMS/OpenMS/blob/develop/src/openms/include/OpenMS/FORMAT/MzMLFile.h#L118 It will then show up in the online documentation

please use `@p` for parameters and `@note` for a note.